nrf_security/cmake/symbol_rename.cmake

```diff
@@ -53,21 +53,21 @@ function(library_redefine_symbols backend template)
     COMMAND ${CMAKE_OBJCOPY} 
             --redefine-syms
             ${CMAKE_CURRENT_BINARY_DIR}/symbol_rename_${MBEDTLS_BACKEND_PREFIX}.txt
-            $<TARGET_FILE:mbedcrypto_${MBEDTLS_BACKEND_PREFIX}>
-            ${BACKEND_RENAMED_LIBARY}
-    DEPENDS mbedcrypto_${MBEDTLS_BACKEND_PREFIX}
+            $<TARGET_FILE:${IMAGE}mbedcrypto_${MBEDTLS_BACKEND_PREFIX}>
+            ${IMAGE}${BACKEND_RENAMED_LIBARY}
```

Comment:
Missing corresponding `${IMAGE}` in line 52

Reply:
fixed. updated file was dangling in editor :(